actiontext/test/unit/model_test.rb

```diff
@@ -49,8 +49,22 @@ class ActionText::ModelTest < ActiveSupport::TestCase
     assert_equal "Hello world", message.content.to_plain_text
   end
 
-  test "save body" do
+  test "saving body" do
     message = Message.create(subject: "Greetings", body: "<h1>Hello world</h1>")
     assert_equal "Hello world", message.body.to_plain_text
   end
+
+  test "saving content via nested attributes" do
+    message = Message.create! subject: "Greetings", content: "<h1>Hello world</h1>",
+      review_attributes: { author_name: "Marcia", content: "Nice work!" }
+    assert_equal "Nice work!", message.review.content.to_plain_text
+  end
+
+  test "updating content via nested attributes" do
+    message = Message.create! subject: "Greetings", content: "<h1>Hello world</h1>",
```

Comment:
Hi @georgeclaghorn ,

When I had debugged this issue, nested attributes update was already working when you pass other database columns(in this case `author_name` for `review_attributes `) in nested attributes hash. It was failing when only rich text field was updated via nested attributes. 

Something like this: https://github.com/rails/rails/pull/35430/files#diff-7706ccd89722c2baed5ee7c410e9f65bR71

I think we should add a test case for it. Let me know if I should open up a PR for it?

Reply:
We’re not passing any other review attributes to `update`, and this test fails without the code changes in this PR.